### PR TITLE
DOC: v4.1.0 rel notes

### PR DESCRIPTION
**Pull request by ZLLentz**
## Description

Big tag

## Motivation and Context

Tags are good
Reviewers are everyone listed as a contributor

### CODE REVIEW

File: docs/source/releases.rst [hunk 1]
```diff
@@ -1,6 +1,79 @@
...
+- Added ``include_plus_sign`` option for ``get_status_float``.
+- Perform continuous integration tests with pip-based installs, with
+  dependencies installed from PyPI.
+
+Contributors
+------------
+- cristinasewell
+- klauer
+- mghaly
+- sheppard
```
Comment (klauer) [inline docs/source/releases.rst:73]:
```suggestion
- jsheppard95
```

**Review by cristinasewell — APPROVED** [review]
Thanks @klauer  for fixing the `get_status_float` and adding new features regarding handling the `status_info`  in general, and fixing my previous `format_status_info`  stuff from previous objects - I saw you fixed that a while ago but never mentioned it. ☺️ 
Thanks @ZLLentz  for doing the tag :)

**Review by klauer — APPROVED** [review]
I trust the aggregation script for the remainder of things 👍

File: docs/source/releases.rst (hunk 1 above)
Comment (klauer) [reply docs/source/releases.rst:73]: Actually `mghaly` -> `ghalym` as well...

**Review by jsheppard95 — APPROVED** [review]

File: docs/source/releases.rst (hunk 1 above)
Comment (ZLLentz) [reply docs/source/releases.rst:73]: Good catches!

Comment (ZLLentz) [reply docs/source/releases.rst:73]: Fixing in separate commit for alphabetical order

**Review by ghalym — APPROVED** [review]

**Pull request merged**